lib/dump/mcdisassembler_value.cpp

```diff
@@ -243,7 +244,8 @@ namespace {
     auto create_asm_info (llvm::Target const & target, llvm::MCRegisterInfo const & register_info,
                           llvm::Triple const & triple) {
         using return_type = error_or<asm_info_ptr>;
-        if (asm_info_ptr asm_info{target.createMCAsmInfo (register_info, triple.getTriple ())}) {
+        const llvm::MCTargetOptions MCOptions = InitMCTargetOptionsFromFlags();
```

Comment:
Naming should be lower-case with `const` after the type (to match the pstore convention). I prefer to avoid single-use variables where possible (can we just pass the result of `InitMCTargetOptionsFromFlags()` directly to `createMCAsmInfo()`)?